src/tea_animationframe.res

```diff
@@ -27,17 +29,17 @@ let every = (~key="", tagger) => {
         switch id.contents {
         | None => ()
         | Some(_stillActive) =>
-          let () = id := Some(Web.Window.requestAnimationFrame(onFrame))
+          let () = id := Some(requestCancellableAnimationFrame(onFrame))
```

Comment:
Because cancelAnimationFrame needs "rafId" as an input.
`@val external cancelAnimationFrame: rafId => unit = "cancelAnimationFrame"`
requestAnimationFrame returns a "unit" so  I instead used  requestCancellableAnimationFrame because it returns "rafId"
```
@val external requestAnimationFrame: (float => unit) => unit = "requestAnimationFrame"
@valexternal requestCancellableAnimationFrame: (float => unit) => rafId = "requestAnimationFrame"
```

Reply:
Gotcha, thanks.